test/utils.ts

```diff
@@ -25,3 +24,14 @@ export const toBN = (num: string | number | BigNumber) => {
 export interface SignerWithAddress extends Signer {
   address: string;
 }
+
+export async function seedWallet(
+  walletToFund: any,
```

Comment:
Why do these need to be any? Doesn't Ethers have a type that you can pull in for signers/wallets?

Reply:
Good point will fix